### ISSUE TITLE
Keyboard not working in Android

### ISSUE THREAD

**Issue by abhisheksett** [pzuraq/liquid-tether#77]
We are using liquid-tether version 2.0.3 and we are facing issue with keyboard in Android. We have text boxes inside liquid-tether container and when we focus on the text box, keyboard comes up and immediately goes away. The issue was not there with 1.1.0 version.

**Comment by pzuraq** [comment]
Does the issue appear on any other mobile browsers? Which versions of Android and which web browsers? And how are you focusing the textbox, manually or programmatically?

**Comment by abhisheksett** [comment]
We have created hybrid app using Cordova and tested in 4.4.4 and 5.1.1. We are focusing textbox manually. Surprisingly the behavior differs in different OEM's. For example, the issue is consistent in Samsung, Lenovo devices, whereas it works fine in Redmi devices.

**Comment by pzuraq** [comment]
Different OEMs = slightly different browsers, so it makes sense that this would only pop up on a subset of machines. I don't have those devices so I can't test it out myself, but if you can reproduce in any browser on iOS, or a desktop browser, I'm happy to dive in.

What's weird to me is that this is happening when you _manually_ (i.e. using your finger) touch the element to focus it. Nothing should be able to interfere with that, unless you have some custom logic running on touch events.

**Comment by abhisheksett** [comment]
We have tested the app in iOS and it works fine. We have tried to find the issue and here is our observation:
When we focus into the textbox, the keyboard comes up and it triggers window resize event. The resize event triggers focus out which eventually hides the keyboard. We have no custom logic written. The same textbox component works fine when it's placed outside tether container.

**Comment by pzuraq** [comment]
Interesting. `liquid-tether` doesn't have any logic that fires on resize events, but Tether.js probably does so that might be the root of it. Can you figure out where the resize event is being started?

**Comment by abhisheksett** [comment]
Yes, the resize event gets triggerred from Tether.js. Tether tries to handle resize with a fix which triggers focus out.

**Comment by pzuraq** [comment]
Ok, so it's probably a problem with Tether.js then. Can you checkout `v2.0.1` of this library to see if it still has the issue? I want to see if the bug was introduced recently in Tether or if it's older.

Either way, you'll probably need to submit a fix to Tether.js. Alternatively, you can use the underlying `liquid-wormhole` library which does all of the animation work, but doesn't do positioning. It sounds like you have a full screen modal, which should be very easy to do with some simple CSS and probably doesn't need the full power of Tether.js.

**Comment by abhisheksett** [comment]
Did you mean to try v2.0.2? Because v2.0.1 is not available.

**Comment by pzuraq** [comment]
https://github.com/pzuraq/liquid-tether/commit/40ce878658a4181d1b6dab5b57342ba717d0fa8f
